src/Microsoft.AspNetCore.Razor.Evolution/RazorEngine.cs

```diff
@@ -54,6 +55,10 @@ internal static void AddDefaults(IRazorEngineBuilder builder)
             builder.Phases.Add(new DefaultRazorIRPhase());
             builder.Phases.Add(new DefaultRazorCSharpLoweringPhase());
 
+            // General extensibility
+            builder.Features.Add(new DefaultRazorDirectiveFeature());
+            builder.Features.Add(new DefaultRazorTargetExtensionFeature());
```

Comment:
Adding these by default because practically speaking they are always gonna be there.